sdk/search/Azure.Search.Documents/tests/SearchIndexerClientTests.cs

```diff
@@ -481,9 +481,74 @@ public async Task CrudSkillset()
                 Context = "/document",
             };
 
+            SearchIndexerKnowledgeStoreTableProjectionSelector table1 = new("hotelReviewsDocument")
```

Comment:
@alzimmermsft , this is where I use knowledge store in service request / response.